tests/test_tekton.py

```diff
@@ -411,6 +411,18 @@ def test_start_pipeline(self, openshift,
                 p_run.start_pipeline_run()
             assert len(responses.calls) == 0
 
+    @responses.activate
+    def test_remove_pipeline(self, openshift):
+        p_run = PipelineRun(os=openshift, pipeline_run_name=PIPELINE_RUN_NAME)
+        responses.add(
+            responses.DELETE,
+            f'https://openshift.testing/apis/tekton.dev/v1beta1/namespaces/'
+            f'{TEST_OCP_NAMESPACE}/pipelineruns/{PIPELINE_RUN_NAME}',
+            json={}
+        )
+        p_run.remove_pipeline_run()
```

Comment:
Return value is not tested here.

Reply:
I think cqi meant that this unit test should check that `remove_pipeline_run()` returns the JSON response

Reply:
extended test to check the response